test/helpers/isWebpack5.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const webpack = require('webpack');
+
+const isWebpack5 = webpack.version[0] === '5';
```

Comment:
should this have a try-catch around it, or have all versions of webpack had a `.version` value?

Reply:
yes, webpack4 and webpack5 have `version` function.

Reply:
Oh, didn't see that this is just being used for test files